sample-operators/mysql-schema/src/test/java/io/javaoperatorsdk/operator/sample/MySQLSchemaOperatorNewE2E.java

```diff
@@ -0,0 +1,111 @@
+package io.javaoperatorsdk.operator.sample;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.fabric8.kubernetes.api.model.NamespaceBuilder;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.LocalPortForward;
+import io.javaoperatorsdk.operator.config.runtime.DefaultConfigurationService;
+import io.javaoperatorsdk.operator.junit.OperatorExtension;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class MySQLSchemaOperatorNewE2E {
+
+  KubernetesClient client = new DefaultKubernetesClient();
+
+  final static String MY_SQL_NS = "mysql";
+
+  // To run on minikube, first:
+  // eval $(minikube -p minikube docker-env)
+  // mvn jib:dockerBuild
+  // and run the tests as usual
+
+  @RegisterExtension
+  OperatorExtension operator =
+      OperatorExtension.builder()
+          .withConfigurationService(DefaultConfigurationService.instance())
+          .withReconciler(new MySQLSchemaReconciler(client,
+              new MySQLDbConfig("127.0.0.1", "3306", "root", "password")))
+          .withDeployment(
+              client.load(new FileInputStream("k8s/operator.yaml")).get())
```

Comment:
cool enough, the `yaml` files provided in the `k8s` folder are already good enough to support this workflow seamlessly!

Reply:
I have some ideas on maybe improving on this… but I need to find some time to work on it :/

Reply:
@metacosm this is going to work smoothly with the Quarkus SDK resources generated in the `target/kubernetes` folder.

> I have some ideas on maybe improving on this

do you wanna share?

Reply:
An annotation processor would make it ... but here we are in the test, I think that performance is not critical ...